loader/CMakeLists.txt

```diff
@@ -406,22 +406,19 @@ if(PKG_CONFIG_FOUND)
         list(REMOVE_DUPLICATES PRIVATE_LIBS)
         set(PRIVATE_LIBS "Libs.private: ${PRIVATE_LIBS}")
     endif()
+
     if(WIN32)
         if(MINGW)
             set(VULKAN_LIB_SUFFIX "-1.dll")
         else()
             set(VULKAN_LIB_SUFFIX "-1")
         endif()
-        # Set libdir path as in cmake's FindVulkan.cmake
-        # https://github.com/KhronosGroup/Vulkan-Loader/issues/668
```

Comment:
At one point FindVulkan.cmake didn't consider MinGW in it's search paths. It now does.
https://gitlab.kitware.com/cmake/cmake/-/merge_requests/8256